stories/features/rewards/concepts.tsx

```diff
@@ -312,6 +312,11 @@ storiesOf('Feature Components/Rewards/Concepts/Desktop', module)
 
     const onRefreshPublisher = () => {
       store.set({ refreshingPublisher: !store.state.refreshingPublisher })
+      setTimeout(() => {
```

Comment:
nit: can we just pass a single object with three props to `store.set`?

Reply:
Done.